feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/metadata.textproto

```diff
@@ -0,0 +1,39 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid: "2beaac46-9b7b-49c4-9bde-62ad530aa5c6"
+plan_id: "RT-5.7"
+description: "Aggregate Not Viable All"
+testbed: TESTBED_DUT_ATE_8LINKS
+
+platform_exceptions:  {
+  platform:  {
+    vendor:  NOKIA
+  }
+  deviations:  {
+    explicit_port_speed: true
+    explicit_interface_in_default_vrf: true
+    aggregate_atomic_update: true
+    interface_enabled: true
+    missing_value_for_defaults: true
```

Comment:
Could you please add below deviaiton 
missing_isis_interface_afi_safi_enable: true

Reply:
Done